doc/cephfs/administration.rst

```diff
@@ -384,3 +384,30 @@ This removes a rank from the failed set.
 
 This command resets the file system state to defaults, except for the name and
 pools. Non-zero ranks are saved in the stopped set.
+
+.. toctree::
```

Comment:
Can you also hide these TOCs like we do in `doc/cephfs/index.rst`. Then the TOC doesn't show up at the bottom.

Reply:
batrick: Roger wilco.